src/java/com/rapleaf/jack/util/JackUtility.java

```diff
@@ -5,37 +5,23 @@
 import java.sql.SQLException;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.function.Function;
 
-import com.google.common.base.Function;
 import org.joda.time.DateTime;
 import org.joda.time.format.DateTimeFormat;
 import org.joda.time.format.DateTimeFormatter;
 
-import com.rapleaf.jack.queries.Index;
-
 public final class JackUtility {
 
   public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormat.forPattern("yyyy-MM-dd");
   public static final DateTimeFormatter TIMESTAMP_FORMATTER = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss");
 
-  public static final Map<Class, Function<Long, String>> FORMATTER_FUNCTION_MAP = new HashMap<Class, Function<Long, String>>(2) {{
-    put(java.sql.Date.class, new DateFormatter(DATE_FORMATTER));
-    put(java.sql.Timestamp.class, new DateFormatter(TIMESTAMP_FORMATTER));
-  }};
-
-  public static final Function<Object, Long> LONG_CASTER = new Function<Object, Long>() {
-    @Override
-    public Long apply(final Object value) {
-      return Long.class.cast(value);
-    }
-  };
+  public static final Map<Class<?>, Function<Long, String>> FORMATTER_FUNCTION_MAP = new HashMap<>(2);
```

Comment:
If we're willing to add a dependency on [LiveRamp commons](https://github.com/LiveRamp/commons), we can use `MapBuilder` to avoid the static initializer block.